src/transformers/training_args.py

```diff
@@ -564,6 +578,27 @@ class TrainingArguments:
             The mode to use in `torch.compile`. If set to any value, `torch_compile` will be set to `True`.
 
             Possible choices are `"default"`, `"reduce-overhead"` and `"max-autotune"`.
+            
+        xla_fsdp (`str`, `dict`, *optional*):
+            Use PyTorch/XLA Fully Sharded Data Parallel Training
+
+            For a complete list of options, please see [here](
+            https://github.com/pytorch/xla/blob/master/torch_xla/distributed/fsdp/xla_fully_sharded_data_parallel.py).
+
+            This is an experimental feature and its API may evolve in the future. The value is the location of config
+            file (e.g., `fsdp_config.json`).
+        xla_fsdp_min_num_params (`int`, *optional*, defaults to `0`):
+            Automatically recursively wrap layers with XLA FSDP if they have at least the specified number of parameters. This setting may only be used with xla_fsdp. It is
+            one of two (mutually exclusive) training arguments that determines an auto wrap policy for XLA FSDP; the other is xla_fsdp_transformer_layer_cls_to_wrap.
+        xla_fsdp_transformer_layer_cls_to_wrap (`List[str]`, *optional*):
+            Automatically recursively wrap layers with XLA FSDP if their transformer layer class name matches one in the specified list (case-sensitive). This setting
+            may only be used with xla_fsdp. It is one of two (mutually exclusive) training arguments that determines an auto wrap policy for XLA FSDP; the other is
+            xla_fsdp_min_num_params.
+        xla_fsdp_grad_ckpt (`bool`, *optional*, defaults to `False`):
+            Will use gradient checkpointing over each nested XLA FSDP wrapped layer. This setting can only be used with
+            xla_fsdp when an auto wrapping policy is specified through xla_fsdp_min_num_params or xla_fsdp_transformer_layer_cls_to_wrap.
```

Comment:
Should all be removed.

Reply:
Yes, thanks. This will be removed in the next push.